modules/ROOT/pages/as2/as2-connector-reference.adoc

```diff
@@ -373,7 +375,6 @@ non-repeatable-stream |  Configure to use repeatable streams. |  |
 * ListenerConfig
 
 
-
 [[as2-mdn-listener]]
 === As2 MDN Listener
```

Comment:
AS2  Do a S&R for these. I know it's generated...

Reply:
Fixed where I could.